hadoop-ozone/cli-admin/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/BasicDatanodeInfo.java

```diff
@@ -19,84 +19,70 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import java.util.List;
 import org.apache.hadoop.hdds.protocol.DatanodeDetails;
 import org.apache.hadoop.hdds.protocol.proto.HddsProtos;
 
 /**
  * Represents filtered Datanode information for json use.
  */
-
-public class BasicDatanodeInfoJson {
-  private final String id;
-  private final String hostName;
-  private final String ipAddress;
-  private final List<DatanodeDetails.Port> ports;
-  private final long setupTime;
-  private final int currentVersion;
-  private final HddsProtos.NodeOperationalState persistedOpState;
-  private final HddsProtos.NodeOperationalState opState;
-  private final long persistedOpStateExpiryEpochSec;
-  private final HddsProtos.NodeState healthState;
-  private final boolean decommissioned;
-  private final boolean maintenance;
-  private final int level;
-  private final int cost;
-  private final int numOfLeaves;
-  private final String networkFullPath;
-  private final String networkLocation;
-  private final String networkName;
+@JsonPropertyOrder({
```

Comment:
Can we use `@JsonProperty(index)` on each method instead of declaring an order up front? Labelling each method will be much easier to maintain. Note that the indices can have gaps between them so entries can be inserted in the future without updating all the following indices. For example we may want to leave a gap in the indices between `ipAddress` and `ports` in case other node identifying info is added later.